core/common/src/test/java/alluxio/AlluxioMockUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+public class AlluxioMockUtil {
+
+  /**
+   * Replace `Whitebox.setInternalState` method in PowerMockito.
+   * Set the value of a field using reflection.
+   */
+  public static void setInternalState(Object object, Field field, Object value) {
```

Comment:
`FieldSetter` cannot set private variables (for example `alluxio.master.job.plan.PlanCoordinatorTest`). I tried looking for mockito internal Whitebox implementation but I couldn't find..

Reply:
FieldSetter is in the internal package of Mockito, it's not meant to be depended on by other projects. Instead of rolling our own implementation of reflection here, can we use [FieldUtils](https://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/reflect/FieldUtils.html) from Commons Lang? It's already a dependency of Alluxio.

Reply:
@dbw9580 Please see the current implementation, using `FieldUtils`, but it needs to be used in combination